Bottom: 5301ba181d4306b1ee62e2d99ada16914490097d
Top:    1f0dc64c14a64b86104fbe870b8a5f58f7b1040c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 02:03:31 +0000

Put back layer opacity to its old behaviour

The exact details of this setting aren't right yet,
so revert them so I can build a release.


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 7f7867b..f7cc15b 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1802,7 +1802,7 @@ ghid_draw_everything (BoxTypePtr drawn_area)
   }
 
   /* draw all layers in layerstack order */
-#define FADE_FACTOR 0.6
+#define FADE_FACTOR 1
   number_phys_on_top = max_phys_group - min_phys_group;
   for (i = ngroups - 1; i >= 0; i--) {
     bool is_this_physical = drawn_groups[i] >= min_phys_group &&
